Product/Production/Services/DocumentSubmissionCore/src/test/java/gov/hhs/fha/nhinc/docsubmission/audit/transform/DocSubmissionAuditTransformsTest.java

```diff
@@ -135,20 +128,20 @@ protected String getRemoteHostAddress(Properties webContextProeprties) {
 
             @Override
             protected String getWebServiceUrlFromRemoteObject(NhinTargetSystemType target, String serviceName) {
-                return soapUIEndpoint;
+                return WS_REQUEST_URL;
             }
         };
 
         ProvideAndRegisterDocumentSetRequestType request = createProvideAndRegisterDocumentSetRequestType();
         RegistryResponseType response = new RegistryResponseType();
         AssertionType assertion = createAssertion();
-        LogEventRequestType auditResponse = transforms.transformResponseToAuditMsg(request, response, assertion,
-            createNhinTarget(), NhincConstants.AUDIT_LOG_INBOUND_DIRECTION, NhincConstants.AUDIT_LOG_ENTITY_INTERFACE,
+        LogEventRequestType auditResponse = transforms.transformResponseToAuditMsg(request, response, assertion, null,
+            NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION, NhincConstants.AUDIT_LOG_NHIN_INTERFACE,
             Boolean.FALSE, webContextProperties, NhincConstants.NHINC_XDR_SERVICE_NAME);
         testGetEventIdentificationType(auditResponse, NhincConstants.NHINC_XDR_SERVICE_NAME, Boolean.FALSE);
-        testGetActiveParticipantDestination(auditResponse, Boolean.TRUE, webContextProperties, soapUIEndpoint);
+        testGetActiveParticipantDestination(auditResponse, Boolean.FALSE, webContextProperties, WS_REQUEST_URL);
         testAuditSourceIdentification(auditResponse.getAuditMessage().getAuditSourceIdentification(), assertion);
-        testGetActiveParticipantSource(auditResponse, Boolean.TRUE, webContextProperties, destinationIP);
+        testGetActiveParticipantSource(auditResponse, Boolean.FALSE, webContextProperties, REMOTE_IP);
         assertParticipantObjectIdentification(auditResponse.getAuditMessage());
     }
 
```

Comment:
Can you add error message to be thrown in case of mismatch to be consistent with other TransformsTest classes?
private void assertParticipantObjectIdentification(AuditMessageType auditMsg) {
}